openhtf/util/logs.py

```diff
@@ -77,54 +78,34 @@ def MyPhase(test, helper):
 import six
 
 
-DEFAULT_LEVEL = 'warning'
-DEFAULT_RECORD_VERBOSITY = 'debug'
-DEFAULT_LOGFILE_LEVEL = 'warning'
-QUIET = False
-LOGFILE = None
-
-LEVEL_CHOICES = ['debug', 'info', 'warning', 'error', 'critical']
 ARG_PARSER = argv.ModuleParser()
-ARG_PARSER.add_argument(
-    '--verbosity', default=DEFAULT_LEVEL, choices=LEVEL_CHOICES,
-    action=argv.StoreInModule, target='%s.DEFAULT_LEVEL' % __name__,
-    help='Console log verbosity level (stderr).')
-ARG_PARSER.add_argument(
-    '--test-record-verbosity', default=DEFAULT_RECORD_VERBOSITY, choices=LEVEL_CHOICES,
-    action=argv.StoreInModule, target='%s.DEFAULT_RECORD_VERBOSITY' % __name__,
-    help='Console log verbosity level for test record (not framework logger).')
-ARG_PARSER.add_argument(
-    '--quiet', action=argv.StoreInModule, target='%s.QUIET' % __name__,
-    proxy=argparse._StoreTrueAction, help="Don't output logs to stderr.")
-ARG_PARSER.add_argument(
-    '--log-file', action=argv.StoreInModule, target='%s.LOGFILE' % __name__,
-    help='Filename to output logs to, if any.')
-ARG_PARSER.add_argument(
-    '--log-file-level', default=DEFAULT_LOGFILE_LEVEL, choices=LEVEL_CHOICES,
-    action=argv.StoreInModule, target='%s.DEFAULT_LOGFILE_LEVEL' % __name__,
-    help='Logging verbosity level for log file output.')
+ARG_PARSER.add_argument('-v', action='append_const', const='v')
```

Comment:
Removing the argv.StoreInModule action makes it hard to change the defaults from an external module.